src/main/java/com/endava/grocy/model/Stock.java

```diff
@@ -1,11 +1,13 @@
 package com.endava.grocy.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import lombok.Data;
 
 @Data
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
This is normally dangerous in testing, it should not be here when we finish the work